source/connection-monitoring-and-pooling/tests/pool-close.yml

```diff
@@ -0,0 +1,10 @@
+version: 1
+style: unit
+description: A Pool MUST be able to be manually closed.
+operations:
+  - name: close
+events:
+  - type: ConnectionPoolCreated
+    address: "42"
+  - type: ConnectionPoolClosed
+    address: "42"
```

Comment:
It might be worth having an empty `ignore` array here to make things easier for statically-typed languages